templates/wiki/compare_revision.html

```diff
@@ -0,0 +1,13 @@
+{% extends "wiki/base.html" %}
+{% block title %}Wiki | Comparing {{ title }}{% endblock %}
+{% block og_title %}Wiki | Comparing {{ title }}{% endblock %}
+{% block og_description %}{% endblock %}
+{% block content %}
+    <div class="uk-container uk-container-small">
+        <h2 class="uk-title">
+            Revision comparison for {{ title }}
+        </h2>
+
+        {{ diff|safe }}
```

Comment:
We put spaces around our operators in Python

Reply:
Fixed in latest commit.